ERCS/erc-201.md

```diff
@@ -0,0 +1,147 @@
+---
+eip: 201
+title: Token Standard
+author: Amitesh Gautam <blockchaindev.amitesh@gmail.com>
+type: Standards Track
```

Comment:
```suggestion
discussions-to: 
type: Standards Track
```
Please create a topic to discuss this ERC in Eth Magicians.  It just needs to include a link to this PR.

Reply:
I am unable to create a topic to discuss in Eth Magicians as I am a new user and the option is not available could you please guide me throught it.

Reply:
To crease a discussions topic you need to read other ERCs for a while.  If you get stuck, please ping me with your username and I can look into it.

Reply:
Thanks for the resolution.